### DISCUSSION TITLE
March Melbourne MicroPython Meetup

### DISCUSSION THREAD

**Discussion by mattytrentini** [micropython/micropython#11072] [General]
Just a heads-up that the [March Melbourne MicroPython Meetup](https://www.meetup.com/micropython-meetup/events/291791261/) is on this Wednesday!

The meetup is held in Melbourne, Australia but will be [live-streamed on zoom](https://planetinnovation.zoom.us/j/98309926039?pwd=S3FZRHNLSzRHMkRFd0htWE4vOHdKUT09) and recorded. 

Last month's videos can be found in discussion #10922. The [work-in-progress slides for the News Roundup](https://docs.google.com/presentation/d/e/2PACX-1vRS0SJE0zdLK0jolkyyO2bYPGd5x9dxop8SWe6vMNsNLfb9qxEVeSrYcBYpI_gmf6sxnWkdEtNR-IQ_/pub) are available - but I'll typically work on them right up until they're being presented!

**Comment by mattytrentini** [comment]
The March videos are available now:

* [Damien covering v1.20 and BLE on the Pico W](https://www.youtube.com/watch?v=fH33D6ER7ls)
* [Me giving the usual News Roundup](https://www.youtube.com/watch?v=DyRCR5ALp38)
  * And the [March News Roundup slides](https://docs.google.com/presentation/d/e/2PACX-1vRS0SJE0zdLK0jolkyyO2bYPGd5x9dxop8SWe6vMNsNLfb9qxEVeSrYcBYpI_gmf6sxnWkdEtNR-IQ_/pub)

Enjoy!

**Reply by robert-hh** [reply]
Thanks for providing the videos and of course for hosting the meeting.
